shared/cluster.go

```diff
@@ -277,6 +277,16 @@ func SonobuoyMixedOS(action, version string) error {
 	return err
 }
 
+// PrintClusterState prints the output of kubectl get nodes,pods -A -o wide
+func PrintClusterState() {
```

Comment:
can u post in here how is the output being like on the console?

Reply:
It's just a flat print, so in the case of suc upgrade it looks like this (grabbed a little bit before and after):

```
PASSED! SUC Upgrade Tests: Verifies DNS Access pre-upgrade
•
 NAME                    STATUS   ROLES                       AGE     VERSION        INTERNAL-IP     EXTERNAL-IP     OS-IMAGE                              KERNEL-VERSION              CONTAINER-RUNTIME
node/ip-172-31-12-103   Ready    control-plane,etcd,master   2m48s   v1.27.7+k3s2   172.31.12.103   a.bbb.ccc.dd    SUSE Linux Enterprise Server 15 SP4   5.14.21-150400.22-default   containerd://1.7.7-k3s1.27
node/ip-172-31-14-118   Ready    <none>                      100s    v1.27.7+k3s2   172.31.14.118   e.fff.gg.hhh    SUSE Linux Enterprise Server 15 SP4   5.14.21-150400.22-default   containerd://1.7.7-k3s1.27
node/ip-172-31-14-52    Ready    control-plane,etcd,master   5m31s   v1.27.7+k3s2   172.31.14.52    ii.jjj.kk.lll   SUSE Linux Enterprise Server 15 SP4   5.14.21-150400.22-default   containerd://1.7.7-k3s1.27
node/ip-172-31-8-184    Ready    control-plane,etcd,master   3m23s   v1.27.7+k3s2   172.31.8.184    mm.nnn.oo.ppp   SUSE Linux Enterprise Server 15 SP4   5.14.21-150400.22-default   containerd://1.7.7-k3s1.27

NAMESPACE        NAME                                         READY   STATUS      RESTARTS   AGE     IP          NODE               NOMINATED NODE   READINESS GATES
dnsutils         pod/dnsutils                                 1/1     Running     0          12s     10.42.3.6   ip-172-31-14-118   <none>           <none>
kube-system      pod/coredns-77ccd57875-gss5h                 1/1     Running     0          5m17s   10.42.0.6   ip-172-31-14-52    <none>           <none>
kube-system      pod/helm-install-traefik-crd-dl5ch           0/1     Completed   0          5m18s   10.42.0.2   ip-172-31-14-52    <none>           <none>
kube-system      pod/helm-install-traefik-k5w8b               0/1     Completed   1          5m18s   10.42.0.4   ip-172-31-14-52    <none>           <none>
kube-system      pod/local-path-provisioner-957fdf8bc-nnh7n   1/1     Running     0          5m17s   10.42.0.5   ip-172-31-14-52    <none>           <none>
kube-system      pod/metrics-server-648b5df564-nkc45          1/1     Running     0          5m17s   10.42.0.3   ip-172-31-14-52    <none>           <none>
kube-system      pod/svclb-traefik-67d50b56-6jm9d             2/2     Running     0          4m26s   10.42.0.7   ip-172-31-14-52    <none>           <none>
kube-system      pod/svclb-traefik-67d50b56-dk67n             2/2     Running     0          3m23s   10.42.1.2   ip-172-31-8-184    <none>           <none>
kube-system      pod/svclb-traefik-67d50b56-tcv8w             2/2     Running     0          100s    10.42.3.2   ip-172-31-14-118   <none>           <none>
kube-system      pod/svclb-traefik-67d50b56-xhkjk             2/2     Running     0          2m47s   10.42.2.2   ip-172-31-12-103   <none>           <none>
kube-system      pod/traefik-768bdcdcdd-n2lpm                 1/1     Running     0          4m26s   10.42.0.8   ip-172-31-14-52    <none>           <none>
test-clusterip   pod/test-clusterip-64d4dcfdd4-6vb77          1/1     Running     0          77s     10.42.3.3   ip-172-31-14-118   <none>           <none>
test-clusterip   pod/test-clusterip-64d4dcfdd4-wr82v          1/1     Running     0          77s     10.42.2.3   ip-172-31-12-103   <none>           <none>
test-daemonset   pod/test-daemonset-2wc8j                     1/1     Running     0          22s     10.42.3.5   ip-172-31-14-118   <none>           <none>
test-daemonset   pod/test-daemonset-fgjn8                     1/1     Running     0          22s     10.42.1.5   ip-172-31-8-184    <none>           <none>
test-daemonset   pod/test-daemonset-x4rf8                     1/1     Running     0          22s     10.42.0.9   ip-172-31-14-52    <none>           <none>
test-daemonset   pod/test-daemonset-xcttm                     1/1     Running     0          22s     10.42.2.5   ip-172-31-12-103   <none>           <none>
test-ingress     pod/test-ingress-469f7                       1/1     Running     0          32s     10.42.1.4   ip-172-31-8-184    <none>           <none>
test-ingress     pod/test-ingress-rzvkk                       1/1     Running     0          32s     10.42.3.4   ip-172-31-14-118   <none>           <none>
test-nodeport    pod/test-nodeport-5b846cdb6d-gmv8x           1/1     Running     0          53s     10.42.1.3   ip-172-31-8-184    <none>           <none>
test-nodeport    pod/test-nodeport-5b846cdb6d-w5lvm           1/1     Running     0          53s     10.42.2.4   ip-172-31-12-103   <none>           <none>


Upgrading cluster to: 

Applying  suc.yaml
```